src/components/OpenWith/OpenWithDropdownMenuItem.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @flow
+ * @file Open With dropdown menu item
+ * @author Box
+ */
+
+import * as React from 'react';
+import { injectIntl } from 'react-intl';
+import MenuItem from 'box-react-ui/lib/components/menu/MenuItem';
+import IconFileDefault from 'box-react-ui/lib/icons/file/IconFileDefault';
+import ICON_FILE_MAP from './IconFileMap';
+
+type Props = {
+    integration: Integration,
+    onClick: Function,
+};
+
+const OpenWithDropdownMenuItem = ({
+    integration: { displayName, displayDescription },
+    onClick,
+}: Props) => {
+    const Icon = ICON_FILE_MAP[displayName] || IconFileDefault;
+    return (
+        <MenuItem onClick={onClick}>
+            <Icon />
+            <span>
```

Comment:
Try to avoid markup that doesn't have semantic value.

Reply:
Based on this layout, I think we need a container so that the p tags don't appear in line. The container that holds the text and the icon is using flexbox with default row direction.